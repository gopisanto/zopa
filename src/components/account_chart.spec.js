import React from 'react';
import renderer from 'react-test-renderer';

import AccountChart from './account_chart';

describe('AccountChart component', () => {
  test('should render correctly based on totalSent and totalLeft amount.', () => {
    const component = renderer.create(
      <AccountChart totalLeft={98000} totalSent={2000} />
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
})
