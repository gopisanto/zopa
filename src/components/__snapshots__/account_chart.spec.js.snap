// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountChart component should render correctly based on totalSent and totalLeft amount. 1`] = `
<div
  className="account-chart"
>
  <svg
    viewBox="0 0 42 42"
  >
    <circle
      cx={21}
      cy={21}
      fill="none"
      r={15.915494309189533}
      stroke="#ECE3EB"
      strokeDasharray="2 98"
      strokeDashoffset={0}
      strokeWidth={10}
    />
    <circle
      cx={21}
      cy={21}
      fill="none"
      r={15.915494309189533}
      stroke="#FDB424"
      strokeDasharray="98 2"
      strokeDashoffset={98}
      strokeWidth={10}
    />
  </svg>
</div>
`;
